static/js/containers/pages/checkout/OrderHistory.js

```diff
@@ -97,9 +127,34 @@ export class OrderHistory extends React.Component<Props> {
               : null}
             <div className="row d-flex p-3 mb-4 bg-light border-top border-2 border-dark">
               <div className="col">
+                Page {this.offset + 1} of{" "}
+                {orderHistory ? Math.ceil(orderHistory.count / 10) : 0} |{" "}
                 {orderHistory ? orderHistory.count : "0"} orders total
               </div>
               <div className="col text-right" />
+              {orderHistory && orderHistory.previous !== null ? (
```

Comment:
could you make this a function that take the "next" or "previous"

Reply:
Moved these to functions (and fixed a code error) in https://github.com/mitodl/mitxonline/commit/06c0734c53e70174b15ceb3cbe7ccaff82418dcb